packages/ng/skeleton/skeleton-button/skeleton-button.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="skeleton is-loading" [class.mod-dark]="dark">
+	<div class="button" [class.mod-dark]="dark" inert>
```

Comment:
I would move the `inert` to the root element of the component but it's not a big deal.